WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -447,22 +447,23 @@ protected override void OnNavigatedTo(bool inHistory, CompositeDisposable dispos
 			PriorLabels.AddRange(_owner.Wallet
 				.KeyManager
 				.GetLabels()
-				.Select(x=>x.ToString()
+				.Select(x => x.ToString()
 					.Split(',', StringSplitOptions.TrimEntries | StringSplitOptions.RemoveEmptyEntries))
-					.SelectMany(x=>x));
+					.SelectMany(x => x));
 
 			PriorLabels = new ObservableCollection<string>(PriorLabels.Distinct());
 
-			_owner.Wallet.Synchronizer.WhenAnyValue(x => x.AllFeeEstimate)
-				.Where(x => x is { })
-				.Select(x => x!.Estimations)
+			var feeProvider = _owner.Wallet.FeeProvider;
```

Comment:
This is a bug fix in the UI. @danwalmsley the point here is that we need to use the FeeProvider's fees and not the Synchronizer's fees. The FeeProvider is smarter, because it also takes fees from Bitcoin Core when it's there and it always gives you the best fees.